cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaPool.java

```diff
@@ -325,6 +325,15 @@ public Set<Integer> scaledDownNodes() {
         return idAssignment.toBeRemoved();
     }
 
+    /**
+     * Generates set of Kafka node IDs going to be added to the Kafka cluster.
```

Comment:
A nit, but I think neither here not in the `currentNodes` method below you don't generate anything. You just get it.

Reply:
Actually the `currentNodes` is not used anymore so I am going to delete it. Regarding the "Generates" word, I was just sticking with what is already used across the other methods which I guess aren't generating nothing as well, or?

Reply:
They are not generating anything neither. But even if you keep them as they are, there is no reason to copy the wrong things from them.

Reply:
Agreed. I fixed the other ones as well.